### PR TITLE
Amls 5842 - Accessibility statement + footer link + toggle

### PR DESCRIPTION
**Pull request by matthewharris1**
This is linked to the frontend repo - https://github.com/hmrc/amls-frontend/pull/1477

## How Has This Been Tested?
manually

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -84,12 +84,17 @@ microservice {
...
+        accessibility-statement = true
+      }
+
       amls-frontend {
         protocol = http
         host = localhost
         port = 9222
         appName: "anti-money-laundering"
         url: ${microservice.services.amls-frontend.protocol}"://"${microservice.services.amls-frontend.host}":"${microservice.services.amls-frontend.port}"/"${microservice.services.amls-frontend.appName}
+        url-accessibility-statement: ${microservice.services.amls-frontend.protocol}"://"${microservice.services.amls-frontend.host}":"${microservice.services.amls-frontend.port}"/"${microservice.services.amls-frontend.appName}"/accessibility-statement"
```
Comment (ajchilts) [inline conf/application.conf:97]: We will need to raise a PR to add this to app-config-base

File: conf/application.conf [hunk 2]
```diff
@@ -84,12 +84,17 @@ microservice {
         welsh-translation: false
       }
 
+      feature-toggle {
```
Comment (ajchilts) [inline conf/application.conf:87]: We will need to add this to app-config-base

File: conf/application.conf (hunk 1 above)
Comment (slavvco) [inline conf/application.conf:97]: if you could change above to:
        url-accessibility-statement: ${microservice.services.amls-frontend.url}"/accessibility-statement"
would be nice :)

Comment (matthewharris1) [reply conf/application.conf]: Done and on EAB

**Pull request merged**